### PR TITLE
Primary/wave/wave UI fif v3

### PR DESCRIPTION
**Pull request by rafifpriyo**
- Add Wave Manager Autoload, Global Class
  - For managing wave attribute. e.g. multiplier score, normal or boss wave
- Add Wave Timer for normal wave
- Add Simple UI for displaying WaveNumber, Timeleft for next wave.

Note: Am i right doing the ```git pull --rebase```

### CODE REVIEW

File: scenes/entities/test-entity.tscn
Comment (LinusAWZZL) [inline scenes/entities/test-entity.tscn:1]: Merge Conflict ni boy

File: scenes/tests/3d-move.tscn [hunk 1]
```diff
@@ -1,17 +1,28 @@
-[gd_scene load_steps=3 format=2]
+<<<<<<< HEAD
```
Comment (LinusAWZZL) [inline scenes/tests/3d-move.tscn:1]: Merge Conflict boy, please resolve

Comment (LinusAWZZL) [reply scenes/tests/3d-move.tscn:1]: contact if you need help

File: scenes/tests/3d-view.tscn [hunk 2]
```diff
@@ -1,17 +1,33 @@
-[gd_scene load_steps=3 format=2]
+<<<<<<< HEAD
```
Comment (LinusAWZZL) [inline scenes/tests/3d-view.tscn:1]: Merge Conflict boy, please resolve
contact if you need help

File: scenes/ui/UI.tscn [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<<<<<<< HEAD
```
Comment (LinusAWZZL) [inline scenes/ui/UI.tscn:1]: Merge Conflict boy, please resolve
contact if you need help

**Review by LinusAWZZL — CHANGES_REQUESTED** [review]
Several small Merge conflicts to be resolved, please resolve before Approval

File: scenes/tests/3d-view.tscn (hunk 2 above)
Comment (rafifpriyo) [reply scenes/tests/3d-view.tscn:1]: Oh yeah, miss that one

File: scenes/ui/UI.tscn (hunk 3 above)
Comment (rafifpriyo) [reply scenes/ui/UI.tscn]: Oh yeah, you are right. Miss that one. No notification from IDE

File: scenes/entities/test-entity.tscn
Comment (rafifpriyo) [reply scenes/entities/test-entity.tscn:1]: Oh yeah, you are right

**Comment by LinusAWZZL** [comment]
I'll merge this ya

**Pull request merged**